internal/tunnel/client/session.go

```diff
@@ -67,6 +67,10 @@ type Session interface {
 	// Latency updates
 	Latency() <-chan time.Duration
 
+	// Close the tunnel with this clientID, with an error that will be reported
+	// from the tunnel's Accept() method.
+	CloseTunnel(clientID string, err error) error
```

Comment:
Let's not make this part of the public interface. I think I saw some SDK feedback that we should remove the other similiar ones from rust/node/python.

Reply:
Like you mention below, left this `CloseTunnel` since it's on the internal client package.